learn/educators/codetour.md

```diff
@@ -9,10 +9,13 @@ MetaDescription: Visual Studio CodeTour Extension
 ---
 # CodeTour
 
+Helping students understand a new codebase can be challenging and time consuming. Creating descriptive enough instructions can be cumbersome and still not easy enough for students to follow. CodeTour can help you introduce your students to your assignments and projects in a streamlined and interactive way.
+
 ## What is CodeTour?
 
 [CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) is a Visual Studio Code extension that allows you to create a step-by-step guide through your codebases. This can be helpful in your classroom to break up your assignments into steps for students to follow or to get them familiar with all parts of the assignment by pointing them to relevant pieces within the assignment.
 
+In the short video below, an instructor is updating her class project to include CodeTour for part 1 of a game development project. They are removing their instructions in code and adding a friendlier CodeTour experience for their students.
```

Comment:
Should we go with "updating their project"?
How about "They are removing existing instructions embedded in the source code and adding..."